### ISSUE TITLE
possible EventEmitter memory

### ISSUE THREAD

**Issue by asm0dey** [zont/gulp-usemin#93]
With latest version I get tons of error like this:
```
Trace
    at Transform.EventEmitter.addListener (events.js:160:15)
    at Transform.Readable.on (/home/finkel/work_sunrise/repos/smev_admin_panel/controller/src/main/webapp/node_modules/gulp-usemin/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:729:33)
    at Transform.EventEmitter.once (events.js:185:8)
    at Transform.Readable.pipe (/home/finkel/work_sunrise/repos/smev_admin_panel/controller/src/main/webapp/node_modules/gulp-usemin/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:576:8)
    at /home/finkel/work_sunrise/repos/smev_admin_panel/controller/src/main/webapp/node_modules/gulp-usemin/lib/pipeline.js:26:27
    at Array.forEach (native)
    at module.exports (/home/finkel/work_sunrise/repos/smev_admin_panel/controller/src/main/webapp/node_modules/gulp-usemin/lib/pipeline.js:25:9)
    at /home/finkel/work_sunrise/repos/smev_admin_panel/controller/src/main/webapp/node_modules/gulp-usemin/lib/htmlBuilder.js:46:5
    at _fulfilled (/home/finkel/work_sunrise/repos/smev_admin_panel/controller/src/main/webapp/node_modules/gulp-usemin/node_modules/q/q.js:794:54)
    at self.promiseDispatch.done (/home/finkel/work_sunrise/repos/smev_admin_panel/controller/src/main/webapp/node_modules/gulp-usemin/node_modules/q/q.js:823:30)
(node) warning: possible EventEmitter memory leak detected. 11 listeners added. Use emitter.setMaxListeners() to increase limit.
```
My setup:
```
$ node -v
v0.10.25
$ npm -v
2.1.16
$ gulp -v
[21:16:59] CLI version 3.8.10
[21:16:59] Local version 3.8.10
$ cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=14.10
DISTRIB_CODENAME=utopic
DISTRIB_DESCRIPTION="Ubuntu 14.10"
$ uname -a
Linux finkel-laptop 3.16.0-29-generic #39-Ubuntu SMP Mon Dec 15 22:27:29 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by DanielTate** [comment]
I have the same issue

(node) warning: possible EventEmitter memory leak detected. 11 listeners added. Use emitter.setMaxListeners() to increase limit.
Trace
    at Stream.addListener (events.js:160:15)
    at Stream.pipe (stream.js:98:10)
    at /Users/frontend7/Projects/Internal/frontend-website/node_modules/gulp-usemin/lib/pipeline.js:26:27
    at Array.forEach (native)
    at module.exports (/Users/frontend7/Projects/Internal/frontend-website/node_modules/gulp-usemin/lib/pipeline.js:25:9)
    at /Users/frontend7/Projects/Internal/frontend-website/node_modules/gulp-usemin/lib/htmlBuilder.js:27:9
    at Function.promise (/Users/frontend7/Projects/Internal/frontend-website/node_modules/gulp-usemin/node_modules/q/q.js:642:9)
    at /Users/frontend7/Projects/Internal/frontend-website/node_modules/gulp-usemin/lib/htmlBuilder.js:21:14
    at Array.map (native)
    at module.exports (/Users/frontend7/Projects/Internal/frontend-website/node_modules/gulp-usemin/lib/htmlBuilder.js:20:25)

➜  ~  node -v
v0.10.33
➜  ~  npm -v
2.0.0
➜  ~  gulp -v
[11:20:07] CLI version 3.8.10

**Comment by Pleasurazy** [comment]
+1

I want to use multi build at same time same html file
like
```
<!-- build:css libraries.css -->files<!-- endbuild -->
<!-- build:css libraries2.css -->files<!-- endbuild -->
<!-- build:css libraries3.css -->files<!-- endbuild -->
```

and a warning shown: 
```
(node) warning: possible EventEmitter memory leak detected. 11 listeners added. Use emitter.setMaxListeners() to increase limit.
Trace:
  at Stream.addListener (events.js:160:15)
  at Stream.pipe (stream.js:71:12)
  at /usr/share/nginx/www/stage.mypoint.tw/op-app/node_modules/gulp-usemin/lib/pipeline.js:26:27
  at Array.forEach (native)
  at module.exports (/usr/share/nginx/www/stage.mypoint.tw/op-app/node_modules/gulp-usemin/lib/pipeline.js:25:9)
  at /usr/share/nginx/www/stage.mypoint.tw/op-app/node_modules/gulp-usemin/lib/htmlBuilder.js:27:9
  at Function.promise (/usr/share/nginx/www/stage.mypoint.tw/op-app/node_modules/gulp-usemin/node_modules/q/q.js:642:9)
  at /usr/share/nginx/www/stage.mypoint.tw/op-app/node_modules/gulp-usemin/lib/htmlBuilder.js:21:14
  at Array.map (native)
  at module.exports (/usr/share/nginx/www/stage.mypoint.tw/op-app/node_modules/gulp-usemin/lib/htmlBuilder.js:20:25)
  at Transform._transform (/usr/share/nginx/www/stage.mypoint.tw/op-app/node_modules/gulp-usemin/index.js:17:9)
  at Transform._read (/usr/share/nginx/www/stage.mypoint.tw/op-app/node_modules/gulp-usemin/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:184:10)
...
...
```

I tried setting emitter.setMaxListeners(0) at gulpfile and it still not work.

if I use single build like
```
<!-- build:css libraries.css -->files<!-- endbuild -->
```

My information

`"gulp-usemin": "^0.3.8",`

```coffee
# gulpfile.coffee
gulp.src 'index.html'
  .pipe plugins.usemin
    js: [plugins.uglify(), 'concat', plugins.rev()]
    css: [plugins.minifyCss(), 'concat', plugins.rev()]
    html: [plugins.minifyHtml(empty:yes)]
  .pipe gulp.dest '.'
```

linode:

```
$ node -v
v0.10.33

$ npm -v
1.4.28

$ gulp -v
[03:43:38] Requiring external module coffee-script/register
[03:43:38] CLI version 3.8.10
[03:43:38] Local version 3.8.10

$ cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=12.04
DISTRIB_CODENAME=precise
DISTRIB_DESCRIPTION="Ubuntu 12.04.5 LTS"
```

index.html

```html
<!DOCTYPE html>
<html id="ngApp" ng-app="opApp">
  <head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    <meta name="viewport" content="width=768, initial-scale=1">
    <!-- build:css libraries.css -->
    <link href="bower_components/angular/angular-csp.css" rel="stylesheet">
    <!-- endbuild -->

    <link href="bower_components/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
    <link href="bower_components/font-awesome/css/font-awesome.css" rel="stylesheet">

    <!-- build:css all.css -->
    <link href="app/blocks/loginForm/main.css" rel="stylesheet">
    <link href="app/layout/main.css" rel="stylesheet">
    <!-- endbuild -->

    <title>{{$root.appTitle}}</title>
  </head>
  <body>
    <header ng-include="'blocks/topNavbar/main.html'"></header>
    <main ng-view class="container"> </main>
    <footer></footer>

    <!-- build:js libraries.js -->
    <script src="bower_components/lodash/dist/lodash.min.js"></script>
    <script src="bower_components/jquery/dist/jquery.min.js"></script>
    <script src="bower_components/angular/angular.min.js"></script>
    <!-- endbuild -->

    <!-- build:js libraries2.js -->
    <script src="bower_components/angular-cookies/angular-cookies.min.js"></script>
    <script src="bower_components/angular-resource/angular-resource.min.js"></script>
    <script src="bower_components/angular-messages/angular-messages.min.js"></script>
    <script src="bower_components/angular-route/angular-route.min.js"></script>
    <script src="bower_components/bootstrap/dist/js/bootstrap.min.js"></script>
    <!-- endbuild -->

    <!-- build:js templates.js -->
    <script src="app/blocks/templates.js"></script>
    <script src="app/views/templates.js"></script>
    <!-- endbuild -->

    <script>(function() {angular.module('opApp', ['opApp.blocks.templates', 'opApp.views.templates', 'ngResource', 'ngRoute', 'ngCookies', 'ngMessages']); }).call(this); </script>

    <!-- build:js all.js -->
    <script src="app/blocks/loginForm/main.js"></script>
    <script src="app/blocks/pagination/main.js"></script>
    <script src="app/blocks/reports/panel.js"></script>
    <script src="app/blocks/search/panel.js"></script>
    <script src="app/blocks/topNavbar/main.js"></script>
    <script src="app/layout/main.js"></script>
    <script src="app/services/api/v3.js"></script>
    <script src="app/services/user/main.js"></script>
    <script src="app/views/me/main.js"></script>
    <!-- endbuild -->
  </body>
</html>
```



-----

btw, localhost (Macbook Pro 2013 Later) is ok

```
ᐅ node -v
v0.10.35

ᐅ npm -v
1.4.28

ᐅ gulp -v
[13:02:17] Requiring external module coffee-script/register
[13:02:17] CLI version 3.8.10
[13:02:17] Local version 3.8.10
```

**Comment by zont** [comment]
Fixed in 0.3.14
